src/ccall.cpp

```diff
@@ -246,8 +247,8 @@ static Value *emit_plt(FunctionType *functype, const AttributeSet &attrs,
                                         llvmgv, runtime_lib);
         builder.CreateStore(builder.CreateBitCast(ptr, T_pvoidfunc), got);
         SmallVector<Value*, 16> args;
-        for (auto &arg: plt->args())
-            args.push_back(&arg);
+        for (Function::arg_iterator arg = plt->arg_begin(), arg_e = plt->arg_end(); arg != arg_e; ++arg)
+            args.push_back(&*arg);
```

Comment:
Not really, but I wanted to keep the code consistent, since the only bit missing is musttail.